src/test/java/com/acrolinx/sidebar/pojo/document/CheckContentTest.java

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2018-present Acrolinx GmbH */
+package com.acrolinx.sidebar.pojo.document;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
please configure your IDE to not use Wildcard imports (on all projects)

(same in a few other classes below)